src/components/EditableFieldView.test.tsx

```diff
@@ -144,4 +179,22 @@ describe('Plain text field in EditableFieldView', () => {
         );
       });
   });
+  it('pending save disables save and cancel buttons', () => {
```

Comment:
I also wanted to verify that the text input was disabled, but I couldn't figure out how to target Cypress appropriately. Suggestions welcome!

Reply:
I'm not sure if this is the best way, but looks like adding this check works:
`cy.get('[data-cy=phs-id-text-field]').children().children('input').should('be.disabled');`

Reply:
Thank you for playing with this, I'll give that a shot!

Reply:
I was able to get this a little tighter, but this suggestion got me on the right track :)